.github/workflows/trigger-load-tests.yml

```diff
@@ -0,0 +1,57 @@
+name: Trigger Load Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      tests:
+        description: >
+          List of test names. This needs to be filled only if you want to run a specific set of tests. Example: foo,bar
+        required: false
+      clusterName:
+        description: 'Cluster name'
+        default: 'log-perf-cluster-test'
+        required: false
+      dispatchType:
+        description: >
+          Repository dispatch type. Specify the dispatch type in order to trigger the result processing workflow.
```

Comment:
here also. https://github.com/ballerina-platform/module-ballerinax-kafka/blob/master/.github/workflows/trigger-load-tests.yml

Reply:
Updated in a6003f3